packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -441,7 +447,7 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 				local: boolean,
 				localOpMetadata: unknown,
 			) => {
-				this.process(message, local, localOpMetadata);
+				this.process(parseHandles(message, this.serializer), local, localOpMetadata);
```

Comment:
I'm not sure why this would change.  The end-to-end outbound path has the same result, and the inbound path wouldn't change.  We _could_ change it here, but might be better to keep for a follow-up.  In the PR as it is, I feel like `processCore` in each DDS would need to be updated correspondingly to this change.

Reply:
since it is idempotent this doesn't hurt, and it would be best if dds didn't need to worry about it at all, its weird to say they need to decode but shouldn't encode